cookbooks/kinematically_driven_subduction_2d/doc/kinematically_driven_subduction_2d.md

```diff
@@ -1,14 +1,14 @@
 # Kinematically-driven 2d oceanic subduction
 
-*This section was contributed by Anne Glerum.*
+*This section was contributed by Anne Glerum and Yingying Li.*
 
 This subduction example is based on the benchmark effort of Quinquis et al.,
 of which initial results were published in {cite:t}`quinquis:2014`. In four
-increasingly complex setups we will go from isoviscous materials without any
+increasingly complex cases we will go from isoviscous materials without any
 temperature effects to a fully thermo-mechanical, nonlinear, strain-weakened
-visco-plastic, externally-driven model of oceanic subduction. The setup of the
-most complex case is outlined in {numref}`fig:QQ_setup`. The models are run for 15 My
-and slab tip depth, trench location, RMS velocity and temperature, and viscous
+visco-plastic, externally-driven model of oceanic subduction. The setup is
+outlined in {numref}`fig:QQ_setup`. The models are run for 15 My
+and slab tip depth, trench location, root-mean-square (RMS) velocity and temperature, and viscous
 dissipation are monitored. In addition, we discuss the effects of the element
 size of the subduction interface and crustal layers, viscosity averaging and
```

Comment:
I don't think you do (yet)

Reply:
Indeed we still don't :D. I wrote that in the Case 1 PR though, so if it was okay then, I guess it's still okay?

Reply:
sure :)